src/Apps/Artwork/Utils/__tests__/createCollectUrl.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { createCollectUrl } from "../createCollectUrl"
+
+jest.mock("sharify", () => ({
+  data: {
+    APP_URL: "https://staging.artsy.net",
+  },
+}))
+
+describe("createCollectUrl", () => {
+  it("formats the collect page url correctly (large)", async () => {
+    const result = createCollectUrl("LARGE", "Painting", "evelyn-walg")
+
+    expect(result).toMatchInlineSnapshot(
+      `"https://staging.artsy.net/collect/painting?page=1&sort=-decayed_merch&acquireable=true&offerable=true&inquireable_only=true&dimension_range=4900-%2A&artist_id=evelyn-walg"`
+    )
+  })
+  it("formats the collect page url correctly (medium)", async () => {
+    const result = createCollectUrl(
+      "MEDIUM",
+      "Video/Film/Animation",
+      "evelyn-walg"
+    )
+
+    expect(result).toMatchInlineSnapshot(
+      `"https://staging.artsy.net/collect/film-slash-video?page=1&sort=-decayed_merch&acquireable=true&offerable=true&inquireable_only=true&dimension_range=1600-4900&artist_id=evelyn-walg"`
+    )
+  })
+
+  it("formats the collect page url correctly (small)", async () => {
+    const result = createCollectUrl(
+      "SMALL",
+      "Drawing, Collage or other Work on Paper",
+      "banksy"
+    )
+
+    expect(result).toMatchInlineSnapshot(
+      `"https://staging.artsy.net/collect/drawing?page=1&sort=-decayed_merch&acquireable=true&offerable=true&inquireable_only=true&dimension_range=0-1600&artist_id=banksy"`
+    )
+  })
+
+  it("formats the collect page url correctly when not filtering by dimension", async () => {
+    const result = createCollectUrl(
+      null,
+      "Drawing, Collage or other Work on Paper",
+      "banksy"
```

Comment:
This is a #nonblocking comment :)

A personal preference of mine is that functions with >= 3 arguments should almost always have those arguments grouped into a single object argument to make code easier to read and modify.

Here's a decent discussion of why https://stackoverflow.com/a/12827204

In this case, to understand the `null` in `createCollectUrl(null, ...)` you need to have in your mind which arguments the function takes and what order they are in. Whereas if this was written `createCollectUrl({size: null, ...})` you get extra information inline to help you understand what the `null` means.